wbooks/src/constants/routes.ts

```diff
@@ -2,3 +2,8 @@ export const ROUTES = {
   BookList: 'BookList',
   BookDetail: 'BookDetail'
 };
+
+export const TABS = {
+  bookList: 'Libros',
+  wishList: 'Lista de deseados'
+};
```

Comment:
This is not correct. Route/Tabs names must be in English:
```
export const TABS = {
  Library: 'Library',
  WishList: 'WishList'
};
```
The tab title must be in option prop